src/components/people.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useContext, useEffect } from 'react';
+import { getPeople } from '../api';
+import { MyContext } from './state';
+import { Loader } from './Loader';
+import { Error } from './error';
+import { Person } from './person';
+import { Person as PersonType } from '../types';
+
+export const People = () => {
+  const { person, setPerson, error, setError, setIsLoading, isLoading } =
+    useContext(MyContext);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(fetchedPeople => {
+        setPerson(fetchedPeople);
+      })
+      .catch(() => {
+        setError('Unable to load people: ');
```

Comment:
The error message 'Unable to load people: ' seems incomplete. Consider providing a full error message for a better user experience.